packages-exp/auth-exp/src/core/strategies/credential.test.ts

```diff
@@ -15,15 +15,18 @@
  * limitations under the License.
  */
 
-import { OperationType, ProviderId, SignInMethod } from '@firebase/auth-types-exp';
 import { expect, use } from 'chai';
 import * as chaiAsPromised from 'chai-as-promised';
+
+import { OperationType, ProviderId, SignInMethod } from '@firebase/auth-types-exp';
```

Comment:
we should make sure we all use the same vscode settings, I feel like they are not consistent across the team

Reply:
We probably should but in this case it's because I forgot to push twice (after the formatter ran)